app/assets/javascripts/controllers/security_group/security_group_form_controller.js

```diff
@@ -12,11 +12,11 @@ ManageIQ.angular.app.controller('securityGroupFormController', ['$http', '$scope
     $scope.securityGroupModel.description = "";
     $scope.newRecord = true;
   } else {
-    miqService.sparkleOn();
```

Comment:
We need the spinner `sparkleOn()` prior to making the API call.
`sparkleOff()` would be called once the data is retrieved.